packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelectMenuItemsWithSearch.tsx

```diff
@@ -104,6 +96,8 @@ export const SingleEntitySelectMenuItemsWithSearch = ({
           {results}
         </>
       )}
+      {entities.entitiesToSelect.length > 0 && <DropdownMenuSeparator />}
+      {createNewButton}
```

Comment:
I didn't wrap the `createNewButton` in a `<DropdownMenuItemsContainer>` although it's done in `MultiRecordSelect`
https://github.com/nicolasrouanne/twenty/blob/58cb1f769aed1053899e98a7599b47cde580fe0e/packages/twenty-front/src/modules/object-record/relation-picker/components/MultiRecordSelect.tsx#L178-L180

I don't understand the use of wrapping a simple button in this container, but maybe I'm missing something

Reply:
I haven't checked out the PR to look at the result yet but I'm guess this might have an impact on styling, probably the padding?
<img width="452" alt="Screenshot 2024-11-12 at 20 38 30" src="https://github.com/user-attachments/assets/fefea0a6-3af0-4895-9975-35f001efbdf9">

Reply:
Hum I might have overlooked that. Will definitely check. Thanks!